src/server/scripts/Northrend/FrozenHalls/ForgeOfSouls/boss_bronjahm.cpp

```diff
@@ -176,6 +201,8 @@ class boss_bronjahm : public CreatureScript
 
                 DoMeleeAttackIfReady();
             }
+        private:
+            uint32 _soulFragmentsSpawned;
```

Comment:
:+1: OK, thanks for the clarification. It is about setting the `uint32 _soulFragmentsSpawned` to zero or some base value, right? So now I just have to Google "how to initialize all fields in the constructor uint32 ". Got any favorite pages about initializing the constructor? :memo: :notebook: